### PR TITLE
ember-attacher migration in partner dashboard

### PR DESCRIPTION
**Pull request by Yibaebi**

### CODE REVIEW

File: app/components/partner/client-report-download/index.hbs [hunk 1]
```diff
@@ -96,7 +99,6 @@
               @clientId={{@clientId}}
               @fileId={{@fileId}}
               @reportId={{this.latestReport.id}}
-              id='password-toggle-{{this.latestReport.id}}'
```
Comment (future-pirate-king) [inline app/components/partner/client-report-download/index.hbs:99]: why is this removed ?

File: app/components/partner/client-report-download/report-password/index.hbs [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
+          {{on 'click' this.togglePassTrayVisibility}}
+          data-test-dropdown-toggle
+        >
+          <AkIcon @iconName='key' />
+        </button>
+      </:default>
+    </AkTooltip>
+
+    <AkPopover
+      @renderInPlace={{true}}
```
Comment (future-pirate-king) [inline app/components/partner/client-report-download/report-password/index.hbs:23]: is `renderInPlace` required ?

File: app/components/partner/client-report-download/report-password/index.scss [hunk 3]
```diff
@@ -2,40 +2,43 @@
...
+
 .dropdown-tray {
-  padding: 0.2rem;
-  width: 16rem;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-}
-.dropdown-content {
+  background-color: var(--common-white);
```
Comment (future-pirate-king) [inline app/components/partner/client-report-download/report-password/index.scss:7]: use `--background-main`

File: app/components/partner/invitation/index.hbs [hunk 4]
```diff
@@ -40,18 +40,22 @@
       title='Resend invite mail'
       {{on 'click' (fn @onResend @request)}}
     >
-      <AkIcon @iconName='refresh' />
-      <AttachTooltip @placemnet='top'>{{t 'resend'}}</AttachTooltip>
+
+      <AkTooltip @placement='top' @arrow={{true}} @title={{t 'resend'}}>
```
Comment (future-pirate-king) [inline app/components/partner/invitation/index.hbs:44]: tooltip should be for button

File: app/components/partner/invitation/index.hbs [hunk 5]
```diff
@@ -40,18 +40,22 @@
...
     <button
       type='button'
       class='is-critical is-icon no-border no-hover-button margin-l-q'
       local-class='delete'
       title='Delete invitation'
       {{on 'click' (fn @onDelete @request)}}
     >
-      <AkIcon @iconName='delete' />
-      <AttachTooltip @placemnet='top'>{{t 'delete'}}</AttachTooltip>
+      <AkTooltip @placement='top' @arrow={{true}} @title={{t 'delete'}}>
```
Comment (future-pirate-king) [inline app/components/partner/invitation/index.hbs:56]: here too tooltip should be for button

**Review by future-pirate-king — CHANGES_REQUESTED** [review]

File: app/components/partner/client-report-download/index.hbs (hunk 1 above)
Comment (Yibaebi) [reply app/components/partner/client-report-download/index.hbs:99]: It wasn't removed entirely. it was relocated [here](https://github.com/appknox/irene/pull/1284/files#diff-0b89776586e7aafa9082f0c376c3946bd69a93416c2b764134531acf1eba1d17R13)

**Review by future-pirate-king — APPROVED** [review]
